lib/misc.c

```diff
@@ -792,3 +792,21 @@ guint b_istr_hash(gconstpointer v)
 
 	return h;
 }
+
+#ifdef NO_STRCASESTR
```

Comment:
isn't there some function in glib that does this that we can use instead? I can't find anything at a first glance in https://people.gnome.org/~ryanl/glib-docs/glib-String-Utility-Functions.html.

Reply:
I did look and did not find anything, and Google does not seem to give me anything useful either.

I can have another look tomorrow, but not sure what it could be named.

Reply:
Didn't find anything at my second glance either.

Reply:
Okay, let's leave that for the moment.